boards/nxp/s32z2xxdc2/s32z2xxdc2_s32z270_rtu1.dts

```diff
@@ -13,9 +13,8 @@
 	compatible = "nxp,s32z270";
 
 	chosen {
-		zephyr,sram = &sram1;
-		zephyr,console = &uart0;
```

Comment:
Is dropping the console here intended ?

Reply:
Hi, yes the change is intended because the console is already chosen in https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/s32z2xxdc2/s32z2xxdc2_s32z270_rtu1_B.overlay (for revB) and https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/s32z2xxdc2/s32z2xxdc2_s32z270_rtu1_D.overlay (for revD)

Reply:
if the change is unrelated to this pr then consider doing it in a separate pr